project/Build.scala

```diff
@@ -1037,7 +1066,7 @@ object Build {
       id = "testAdapter", base = file("test-adapter")
   ).settings(
       commonSettings,
-      publishSettings,
+      publishSettings(VersionScheme.BreakOnSevere),
```

Comment:
I do not think this is correct either: The `testAdapter` has a strict version dependency with the `testBridge`.

Reply:
Right, so it should be `BreakOnMinor`. But then we end up with the same problem as with scalajs-ir, so we have to apply the same workaround as for scalajs-ir.

Reply:
We could also solve this better by putting a `val scalaJSBridgeVersion` inside the `test-adapter` artifact. Then sbt-scalajs would read that version to decide what version of `scalajs-test-bridge` to depend on. That would solve the issue, I believe.